docs/src/components/Docs/Props/CustomizingToolbarProp/defaultToolbar.js

```diff
@@ -104,6 +104,7 @@ export default "{\n" +
   "    component: undefined,\n" +
   "    popupClassName: undefined,\n" +
   "    embedCallback: undefined,\n" +
+  "    embedValidator: undefined,\n" +
```

Comment:
linkValidator field is missing in this file

Reply:
Fixed in 7f61272